torch/onnx/utils.py

```diff
@@ -45,7 +63,7 @@ def set_training(model, mode):
             model.train(old_mode)
 
 
-def export(model, args, f, export_params=True, verbose=False, training=False,
```

Comment:
"training" param already exists, but we do not support exporting fully trainable params. 
Would it be ok to rename this param to something like "training_amenable" or "training_friendly" to avoid user confusion?

Reply:
@houseroad - is the existing `training` parameter used anywhere? Can we consider removing it, so as to avoid confusion?

Reply:
I am not aware of any use case.
`training` is aligned with parameter in pytorch model.

We can turn it into an enum parameter: 1) Eval, 2) Preserve, 3) Train. I prefer to use eval as default.